tests/Extensions/DataGrid.OData.Client/Microsoft.FluentUI.AspNetCore.Components.DataGrid.OData.Client.Tests.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
This **csproj** doesn't contain any files or unit tests.
Can you remove it?

This is the same for the `Readme.md` file

Reply:
@dvoituron my bad! I've removed the empty unit test project, and added content to the Readme.md file much like is in the EF version description... I like keeping things consistent :)